src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -177,6 +177,14 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, string tag,
                         package.RequiresPerVertexColoration = true;
                     }
 
+                    //If the package has a transform that is not the identity matrix
+                    //then set requiresCustomTransform to true.
+                    if (package is ITransformable && (package as ITransformable).Transform.SequenceEqual(
```

Comment:
package as ITransformable is used in if loop and inside the if loop. Can this be moved outside the loop? say in a variable?